"use strict";(self.webpackChunkmy_website=self.webpackChunkmy_website||[]).push([[4139],{5520:e=>{e.exports=JSON.parse('{"label":"Error Handling","permalink":"/main-website/docs/tags/error-handling","allTagsPath":"/main-website/docs/tags","count":1,"items":[{"id":"tuts/tutorials/programming/python/error-handling-and-exception-handling","title":"Error Handling and Exception Handling","description":"Errors and exceptions are an inevitable part of programming. Python provides a robust error handling mechanism to catch and handle these errors gracefully. In this section, we\'ll explore how to handle errors and exceptions effectively in Python.","permalink":"/main-website/docs/tuts/tutorials/programming/python/error-handling-and-exception-handling"}]}')}}]);